cluster/k8s-1.9.3/README.md

```diff
@@ -9,8 +9,8 @@ local machine and are the pushed to a registry which is exposed at
 ## Bringing the cluster up
 
 ```bash
-export PROVIDER=k8s-1.9.3
-export VAGRANT_NUM_NODES=1 # master + one nodes
+export KUBEVIRT_PROVIDER=k8s-1.9.3
+export KUBEVIRT_NUM_NODES=1 # master + one nodes
```

Comment:
since we're shaking things up and devs already have to update their ENV VARS, should we consider having the number of nodes actually represent the number of nodes instead of (num nodes + 1)?

As I remember, i think this was left over from something we had to consider in the VagrantFile

Reply:
Yes, probably makes sense to count schedulable nodes by this. So KUBEVIRT_NUM_NODES would equal to a schedulable master, or a not schedulavble master + one schedulable node, depending on the cluster configuration. That would be consistent.

Reply:
done